Build/Build.cs

```diff
@@ -86,6 +82,11 @@ class Build : NukeBuild
         {
             SemVer = GitVersion.SemVer;
 
+            foreach (var pi in GitVersion.GetType().GetProperties())
+            {
+                Information("{Name} = {Value}", pi.Name, pi.GetValue(GitVersion));
+            }
+
```

Comment:
@dennisdoomen Do you mind if I take this "SemVer"-thing over?

Reply:
I bumped GitVersion to 6.0.2 to see if this fixes the recurring issue of GitVersion not being able to calculate the version. But indeed, it also changed something else.

Reply:
But sure. If you can create a new PR that bumps the GitVersion version to 6.0.2 _and_ maintain the correct version, that would be great.

Reply:
Cool :)

Reply:
For now, I think we cannot rely on GitVersion to give us the correct one, so I think I have to go another way (for now)

Reply:
Have you tried a clean project?

Reply:
No

I actually think that this could be related to NUKE, because of breaking changes in the command response

Reply:
I can't think of anything. Nuke's GitVersion support just exposes all the properties that GitVersion returns.